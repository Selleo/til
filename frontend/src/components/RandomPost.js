import React from 'react'

const RandomPost = () => {
  // TODO fetch random post
  return <p>random post goes here</p>
}

export default RandomPost
