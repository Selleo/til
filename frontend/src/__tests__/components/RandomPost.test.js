import React from "react";
import { render } from "@testing-library/react";
import RandomPost from "../../components/RandomPost";

describe("<RandomPost/>", () => {
  it("renders properly", () => {
    const { asFragment } = render(<RandomPost />);

    expect(asFragment()).toMatchSnapshot();
  });
});
