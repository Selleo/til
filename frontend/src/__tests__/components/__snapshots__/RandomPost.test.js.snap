// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RandomPost/> renders properly 1`] = `
<DocumentFragment>
  <p>
    random post goes here
  </p>
</DocumentFragment>
`;
